tests/test_vsshape.py

```diff
@@ -501,4 +501,4 @@ def test_delete_storage_rule(self):
         s.sendAuthorized = MagicMock(return_value=self.MockedResponse(200, self.test_storage_rule))
 
         s.delete_storage_rule()
-        s.sendAuthorized.assert_called_with('DELETE', '/API/item/VX-456/storage-rule/original', None, {'Accept': 'application/xml'})
+        s.sendAuthorized.assert_called_with('DELETE', '/API/item/VX-456/storage-rule/original', None, {'Accept': 'application/xml'}, rawData=True)
```

Comment:
I don't think we need rawData=True if the content body is xml.

Reply:
Fixed.